Sources/MapboxNavigation/NavigationMapView.swift

```diff
@@ -596,10 +550,7 @@ open class NavigationMapView: UIView {
         pendingCoordinateForRouteLine = nil
     }
     
-    @discardableResult func addRouteRestrictedAreaLayer(_ route: Route,
```

Comment:
I think that it would've been better to land fix for restricted areas position in different PR. It's hard to follow what was exactly wrong and how such issue was fixed.

Reply:
The restricted area position is caused by the line gradient stops generation. When we remove the fractionTraveled and adjust the base color for empty restricted feature gradient stops generation, the issue will be fixed. It's hard to split them to two.